### PR TITLE
Make the User guide compile again

### PR DESCRIPTION
**Pull request by hmenke**
**tl;dr** Don't use underscores in `\label`.

The user guide used to not compile on the first try with an error message like this
````
! Missing \endcsname inserted.
<to be read again> 
                   \unhbox 
l.23 ... on Ubuntu 16.04 LTS}{subsection.1.5.1}{}}
````
This is due to the fact that someone used an underscore in a `\label`.  The user guide does `\usepackage[strings]{underscore}`.  This package will make the underscore an active character, such that you can conveniently type text containing underscores without having to escape them using `\_` while retaining the meaning as the subscript operator in math mode.  Unfortunately, the definition of the macro behind the active underscore is not e-TeX protected which leads to it being expanded once written to an `aux` file (which is what `\label` does).  It then tries to expand assignments somewhere deep down in the expansion chain which is not possible at this point which then leads to the error.

### CODE REVIEW

**Comment by hmenke** [comment]
The only file touched is `introduction.tex`, so the testsuite should not complain.
